lib/versioning/types.ts

```diff
@@ -26,7 +26,7 @@ export interface VersioningApi {
   isLessThanRange?(version: string, range: string): boolean;
   getSatisfyingVersion(versions: string[], range: string): string | null;
   minSatisfyingVersion(versions: string[], range: string): string | null;
-  getNewValue(newValueConfig: NewValueConfig): string;
+  getNewValue(newValueConfig: NewValueConfig): string | null;
```

Comment:
@rarkins Should we ever return null from that function? Or should we return current value?

Reply:
We discussed this  [here](https://github.com/renovatebot/renovate/pull/13278#discussion_r774909803)

Reply:
We discussed this [here ](https://github.com/renovatebot/renovate/pull/13278#discussion_r774909803)too but it was incomplete.

Reply:
It seems to be strictly defined:
https://github.com/renovatebot/renovate/blob/75b1d8fbc60fa6fdd0c6ce602e96ee831109400b/lib/manager/types.ts#L139

IMO, `versioning.getNewValue()` should be able to return null, however we need make sure every `versioning.getNewValue(...)` call in `lib/workers/` directory explicitly replaces null results with `currentValue`. 

Otherwise, we have to stick to convention and return `currentValue` by default from every particular versioning implementation.

Reply:
https://github.com/renovatebot/renovate/pull/13336

Reply:
UPD. This type change should be okay, it seems we're already handling `null` results well